app/lib/publishing_api.rb

```diff
@@ -0,0 +1,9 @@
+class PublishingApi
+  def self.instance
+    GdsApi::PublishingApi.new(
```

Comment:
I think it would be better to be consistent about memo-izing the singleton instance as you did for `RummagerApi` in [this commit][1], i.e. as per the implementation of [`SingletonDependency#get`][2].

[1]: 0aabc2e51d8135b88aad91751bfa7f991f00e3fd
[2]: https://github.com/alphagov/manuals-publisher/blob/0500169731a70ff35b615e1a599637f86d991e11/lib/dependency_container.rb#L82